layouts/html.hbs

```diff
@@ -146,8 +146,8 @@
           } else if (window.isOverlay) {
             window.Overlay.onMessage(message);
           } else if (message.eventType && message.eventType === 'click') {
-            ANSWERS.components._activeComponents
-              .filter(component => component.name === 'SearchBar.autocomplete')
+             ANSWERS.components
+              .getActiveComponents('AutoComplete')
```

Comment:
In the name of writing observations that don't matter in the grand scheme of things, I think closing other AutoComplete usages besides SearchBar ones is the behavior we want! It will probably never come into play since we don't have FilterSearch or the GeolocationFilter (which is different than LocationBias) but it's nice nonetheless!